payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncherContract.kt

```diff
@@ -44,13 +45,22 @@ class GooglePayPaymentMethodLauncherContract :
      * @param config the [GooglePayPaymentMethodLauncher.Config] for this transaction
      * @param currencyCode ISO 4217 alphabetic currency code. (e.g. "USD", "EUR")
      * @param amount if the amount of the transaction is unknown at this time, set to `0`.
+     * @param transactionId a unique ID that identifies a transaction attempt. Merchants may use an
+     *     existing ID or generate a specific one for Google Pay transaction attempts.
+     *     This field is required when you send callbacks to the Google Transaction Events API.
      */
     @Parcelize
     data class Args(
         internal val config: GooglePayPaymentMethodLauncher.Config,
         internal val currencyCode: String,
-        internal val amount: Int
+        internal val amount: Int,
+        internal val transactionId: String? = null,
     ) : Parcelable {
+        constructor(
```

Comment:
In order to stay backwards compatible for Java.

Reply:
use `@JvmOverloads` ?

Reply:
It's not supported here.